app/components/common/eventCard/index.tsx

```diff
@@ -0,0 +1,39 @@
+interface EventCardProps {
+  date: number;
```

Comment:
instead of taking `date`, `month` and `time` as props:
- Take the `full date` as props.
- Create a `util` to and break a `full date` to `date`, `month` and `time`.
- Use that util inside this component.

Reply:
Great suggestion, I have updated my code accordingly.
Thanks!